sway-ast/src/item/mod.rs

```diff
@@ -66,14 +66,20 @@ impl Spanned for ItemKind {
 
 #[derive(Clone, Debug, Serialize)]
 pub struct TypeField {
+    pub visibility: Option<PubToken>,
```

Comment:
Is it likely we'll add other visibility tokens in the future, similar to `pub(crate)`?

Reply:
This might be the case. If it happens, we will need to change the way we define visibility everywhere, starting from the `TypeField` in the AST.